flutter/lib/ui/main_screen.dart

```diff
@@ -133,19 +134,28 @@ class MyHomePage extends StatelessWidget {
 class MyPaintBottom extends CustomPainter {
   @override
   void paint(Canvas canvas, Size size) {
+    final List<Color> colors;
+    if (OFFICIAL_BUILD) {
+      colors = [
+        Color(0xFF31A3E2),
+        Color(0xFF31A3E2),
+        Color(0xFF31A3E2),
+        Color(0xFF3189E2),
+        Color(0xFF0B4A7F),
+      ];
+    } else {
+      colors = [
+        Colors.brown,
+        Colors.brown,
```

Comment:
It think it would be better to use something like `Colors.brown.shade400` as the second color. Blank brown without any gradient doesn't look good with our UI.
Same applies to progress screen.